index.html

```diff
@@ -241,8 +241,9 @@ <h3 class="about-me-item">Skills</h3>
                 </li>
             </ul>
         </section>
-        
-        <footer class="contact-us-footer">
+
+        <footer class="footer-contact-us">
+
             <h3 class="contact-us-text">
```

Comment:
- [ ] [OPTIONAL] You have done a job with the form however, kindly change the `h3` tag into a paragraph tag `p`.